### ISSUE TITLE
Support for defaultProps will be removed error

### ISSUE THREAD

**Issue by Luhari** [wellbees/color-picker-input#3]
Hi, I just added this color picker to my react project and everytime the color picker is rendered, it shows this error. 

> ColorPicker: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.

![electron_18PcTxdV47](https://github.com/user-attachments/assets/61e75657-b4a2-44fa-b347-3e1b3e25e396)

I'm using react ^18.3.1.

Although the message shows it as a warning, the log is done with console.error which causes my unit tests to fail.

**Comment by azizsenturk** [comment]
Hello, thanks for the information. Compatibility issues have been fixed for React 18.3.1 version. You can try again.

**Comment by Luhari** [comment]
Sorry for the delayed feedback, I just updated the package and the warning still appears. I have made sure the package is updated by removing the node_modules and reinstalling with the updated version in the package.json.

Did it work for you? 

The warning is indeed different from last version. Now it shows up specifically at opening the picker.
![image](https://github.com/user-attachments/assets/aa07ffdb-6c72-44c0-bb42-b5d58e7e3d44)

Is it something inside your package or this is something unrelated to the package (like maybe a dependency package)?
